Misc/NEWS.d/next/C API/2023-06-19-20-02-16.gh-issue-105922.o4T6wO.rst

```diff
@@ -0,0 +1,4 @@
+Add :c:func:`PyImport_AddModuleRef` function to replace the deprecated
+:c:func:`PyImport_AddModule` function: return a :term:`strong reference` to
```

Comment:
The deprecation of `PyImport_AddModuleObject` should also be mentioned in the NEWS entry.

Reply:
I changed my mind, I prefer to deprecate the function in a separated PR.